### PR TITLE
Prepare release 1.8

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves https://github.com/jaegertracing/jaeger/issues/1150

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+- Add a metric for number of partitions held ([#1154](https://github.com/jaegertracing/jaeger/pull/1154), [@vprithvi](https://github.com/vprithvi))
+- Refactor agent configuration ([#1092](https://github.com/jaegertracing/jaeger/pull/1092), [@pavolloffay](https://github.com/pavolloffay))
+- Log jaeger-collector tchannel port ([#1136](https://github.com/jaegertracing/jaeger/pull/1136), [@mindaugasrukas](https://github.com/mindaugasrukas))
+- Rename `params.Factory` to `params.MetricFactory` ([#1146](https://github.com/jaegertracing/jaeger/pull/1146), [@vprithvi](https://github.com/vprithvi))
+- Fix child span context while tracing cassandra queries ([#1131](https://github.com/jaegertracing/jaeger/pull/1131), [@annanay25](https://github.com/annanay25))
+- Support tracer env based initialization in hotrod ([#1115](https://github.com/jaegertracing/jaeger/pull/1115), [@eundoosong](https://github.com/eundoosong))
+- Deadlock detector hack for Kafka driver instability ([#1087](https://github.com/jaegertracing/jaeger/pull/1087), [@vprithvi](https://github.com/vprithvi))
```
Comment (yurishkuro) [inline CHANGELOG.md]: maybe we should include #1134. I did a master merge to see if the build is clean.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
 
 #### Backend Changes
 
+- Emit spans for elastic storage backend ([#1128](https://github.com/jaegertracing/jaeger/pull/1128), [@annanay25](https://github.com/annanay25))
+- Allow to use TLS certificates for Elasticsearch authentication ([#1139](https://github.com/jaegertracing/jaeger/pull/1139), [@clyang82](https://github.com/clyang82))
+- Add ingester metrics, healthcheck and rename Kafka cli flags ([#1094](https://github.com/jaegertracing/jaeger/pull/1094), [@ledor473](https://github.com/ledor473))
+- Add a metric for number of partitions held ([#1154](https://github.com/jaegertracing/jaeger/pull/1154), [@vprithvi](https://github.com/vprithvi))
+- Refactor agent configuration ([#1092](https://github.com/jaegertracing/jaeger/pull/1092), [@pavolloffay](https://github.com/pavolloffay))
+- Log jaeger-collector tchannel port ([#1136](https://github.com/jaegertracing/jaeger/pull/1136), [@mindaugasrukas](https://github.com/mindaugasrukas))
+- Rename `params.Factory` to `params.MetricFactory` ([#1146](https://github.com/jaegertracing/jaeger/pull/1146), [@vprithvi](https://github.com/vprithvi))
```
Comment (yurishkuro) [inline CHANGELOG.md]: not relevant for release notes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,43 @@
 Changes by Version
 ==================
 
-1.8.0 (unreleased)
+1.8.0 (2018-11-12)
 ------------------
 
 #### Backend Changes
```
Comment (yurishkuro) [inline CHANGELOG.md]: I think we should split new features & bug fixes, right now it's a mixed bag

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,13 +1,43 @@
...
+- Fix child span context while tracing cassandra queries ([#1131](https://github.com/jaegertracing/jaeger/pull/1131), [@annanay25](https://github.com/annanay25))
+- Support tracer env based initialization in hotrod ([#1115](https://github.com/jaegertracing/jaeger/pull/1115), [@eundoosong](https://github.com/eundoosong))
+- Deadlock detector hack for Kafka driver instability ([#1087](https://github.com/jaegertracing/jaeger/pull/1087), [@vprithvi](https://github.com/vprithvi))
+- Use Go 1.11 ([#1104](https://github.com/jaegertracing/jaeger/pull/1104), [@isaachier](https://github.com/isaachier))
+- Fix processor overriding data in a buffer ([#1099](https://github.com/jaegertracing/jaeger/pull/1099), [@pavolloffay](https://github.com/pavolloffay))
+- Publish ingester as binaries and docker image ([#1086](https://github.com/jaegertracing/jaeger/pull/1086), [@ledor473](https://github.com/ledor473))
+- Tag images with commit SHA and publish to `-snapshot` repository ([#1082](https://github.com/jaegertracing/jaeger/pull/1082), [@pavolloffay](https://github.com/pavolloffay))
+
 ##### Breaking Changes
```
Comment (yurishkuro) [inline CHANGELOG.md]: usually we do breaking changes above the new/fixes sections

**Comment by pavolloffay** [comment]
Can we also get https://github.com/jaegertracing/jaeger-ui/pull/263 resolved and include it in this release? We would like to have it in 1.8

**Comment by yurishkuro** [comment]
I expect the embedded UI PR to still go through a few review iterations for a few days (and Joe is traveling this week, he may not have a lot of time to look at it). I would suggest not blocking on it.

**Comment by objectiser** [comment]
Would we be able to do a patch release once the ui change is ready?

**Comment by yurishkuro** [comment]
yes, but we might want to do it from a branch off of the 1.8 version, since we're planning to merge other breaking PRs right after the release.

**Comment by pavolloffay** [comment]
Cool I wanted to check the same. Then I guess we can go ahead and merge this today/tomorrow. The docs are updated https://github.com/jaegertracing/documentation/pull/167. The only missing part is the architecture diagram which can be done later.

File: CHANGELOG.md (hunk 3 above)
Comment (pavolloffay) [reply CHANGELOG.md]: done I have also unified the titles :scissors:

**Pull request merged**